src/commands.rs

```diff
@@ -589,6 +589,11 @@ implement_commands! {
         cmd("LPOP").arg(key)
     }
 
+    /// Removes and returns up to `count` number of elements from the list stored at `key`
+    fn lpop_bulk<K: ToRedisArgs>(key: K, count: usize) {
```

Comment:
Could you add this as `Option<usize>` to `lpop` instead? We already have some breaking changes going out soon

Reply:
Sounds good